src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -13,7 +13,33 @@
 import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
 
 /**
- * Print reads from a SAM/BAM/CRAM file.
+ * Copy reads in a SAM/BAM/CRAM file verbatim to a SAM file
+ *
+ * <p>
+ *     This tool copies reads from a SAM/BAM/CRAM file to a SAM file. If you are interested in printing reads in
+ *     certain intervals only, use the -L argument.
+ *
+ * </p>
+ *
+ * <h3> Input </h3>
+ * <ul>
+ *     <li> SAM/BAM/CRAM file to be copied to a SAM file </li>
+ *     <li> (optional) interval file with the -L argument </li>
```

Comment:
I wouldn't call an interval file an input.  Yes, it's a file the user supplies, but it's more of an argument.